### ISSUE TITLE
Doesn't work (Win10)

### ISSUE THREAD

**Issue by Hexaae** [phiresky/RobinHood-TheLegendOfSherwood-Resolution-Patcher#1]
```
respatcher.exe 1920 1080
terminate called after throwing an instance of 'std::logic_error'
  what():  basic_string::_S_construct NULL not valid
```

**Comment by phiresky** [comment]
Yeah, the given windows executable can't work without modification because the code doesn't include logic to detect the path of the config file for windows.. I think I just compiled it for Windows to see if it works in theory. You'll have to modify the code to use the appropriate AppData file path on Windows.

I sadly don't actually have the Windows version since the Linux version is sold separately, so I can't really help out.

**Comment by swi-da** [comment]
Sorry to resurrect this old zombie issue :) If you would get the windows version, any chance you can still release a working windows executable?

**Comment by nhanh9d** [comment]
I saw your cpp file and I think it would be best if you allow user to set path as an input. I'm developing another version with another input that allow you to choose path

**Comment by phiresky** [comment]
Feel free to send a PR ;)
